application_handlers.go

```diff
@@ -327,6 +327,13 @@ func ApplicationPause(c echo.Context) error {
 		return err
 	}
 
+	// Check if the application exists
+	_, err = applicationDao.GetById(&applicationId)
```

Comment:
I think we can use `applicationDao.Exists(id)` instead.

Reply:
good point 👍🏻 I forgot we have it